src/wagtail_ai/static_src/image_description/main.tsx

```diff
@@ -0,0 +1,62 @@
+import './main.css';
+import { fetchResponse } from '../api';
+
+import wandIcon from '../wand_icon.svg';
+
+document.addEventListener('wagtail-ai:image-form', (event) => {
+  const input = event.target as HTMLInputElement;
+  const imageId = input.dataset['wagtailaiImageId'];
```

Comment:
**Nitpick:** Given we're not accessing dynamically generated keys, I don't think we should be using `[' ']` syntax. `input.dataset.wagtailaiImageId` looks more appropriate for static values.